### ISSUE TITLE
Connection to server doesn't take the new port

### ISSUE THREAD

**Issue by jtsalten** [intersystems-community/intersystems-servermanager#89]
Issue Type: <b>Bug</b>

Define and use a server to IRIS. Then without closing VS Code, change the port to connect to... let's say you want to start using a new instance of IRIS keeping the same server name. No matter what you do, Toggling, Refreshing connection, VS Code keep trying with the last/old port instead of using the new port associated with that updated server configuration.

Extension version: 2.0.3
VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:35:57.887Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.79GB (3.10GB free)|
|Process Argv|--crash-reporter-id 358469c0-fdbf-4ba7-9019-19e4937e55fb|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt602cf:30294773
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvspyt639:30291489
pythontb:30283811
pythonvspyt551cf:30291415
vspre833cf:30267465
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30286553

```

</details>

**Comment by isc-bsaviano** [comment]
Maybe related to https://github.com/intersystems-community/vscode-objectscript/issues/608

**Comment by jtsalten** [comment]
client-side.... the server entry is at workspace level... coud that be the reason why is not taken into account? I always thought that, if working with workspaces, the best place to define the servers was within that workspace settings... but maybe I'm wrong...

**Comment by gjsjohnmurray** [comment]
> 
> 
> Maybe related to [intersystems-community/vscode-objectscript#608](https://github.com/intersystems-community/vscode-objectscript/issues/608)

@isc-bsaviano is correct. This is a duplicate, and fixed in the upcoming 1.0.11
